css/style.css

```diff
@@ -1,7 +1,316 @@
+html {
+	box-sizing: border-box; /*padding и border включаются в общую ширину и высоту элемента*/
+ }
+
+*, *::before, *::after {
+	box-sizing: inherit;
+}
+
+html. body {
+	height: 100%;
+}
+
+body {
+		margin: 0 auto; 
+	color: #E8EEE4; 
+	font: normal 22px/26px MuseoSansCyrl;
+	overflow: scroll;
+}
+
+.wrap {
+	max-width: 1920px;
+	height: 100vh;
+	background: url(../img/iswanto-arif-448003-unsplash.png) no-repeat center; /*фоновая картинка*/
+	background-size: cover; /*картинка полностью покрывает блок при любых размерах*/
+	margin: 0 auto;
+}
+
+/*header styles start*/
+
+.header {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	padding: 5vh;
+	margin-bottom: 9vh;
+}
+
+.header__logo {
+	margin-left: 5vw;
+}
+
+.header__logo img {
+	max-width: 100%;
+	max-height: 100%;
+}
+
+.header__menu {
+	display: flex;
+	justify-content: flex-end;
+}
+
+.menu__link {
+	flex-wrap: nowrap;
+	color: #E8EEE4;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#E8EEE4" to be "#e8eee4" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Project-Bali/pullRequest?prid=5455736)